lib/ioki/model/operator/passenger_option.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      class PassengerOption < Base
+        attribute :type,
+                  on:   :read,
+                  type: :string
+
+        attribute :id,
+                  on:   :read,
+                  type: :string
+
+        attribute :created_at,
+                  on:   :read,
+                  type: :date_time
+
+        attribute :updated_at,
+                  on:   :read,
+                  type: :date_time
+
+        attribute :slug,
+                  on:   [:read, :create],
+                  type: :string
+
+        attribute :default_value,
+                  on:   [:read, :create, :update],
+                  type: [:string, :boolean, :integer]
+
+        attribute :bookable,
+                  on:   [:read, :create, :update],
+                  type: :boolean
+
+        attribute :data_type,
+                  on:   [:read, :create],
+                  type: :string
+
+        attribute :option_type,
+                  on:   [:read, :create, :update],
+                  type: :string
+
+        attribute :sort_key,
```

Comment:
I don't find the `sort_key` in the serializer? Maybe I'm looking in the wrong place?

Reply:
I forgot to add the `sort_key` in triebwerk. But it's fixed now. Thanks!